### ISSUE TITLE
User Story 9 – Basic Design

### ISSUE THREAD

**Issue by StephanBergmann** [miriam-ertl/my-trips#11]
## Value proposition

**As a** user
**I want to** see a more organized and user-friendly user interface (UI)
**so that** I can easily interact with the application and complete tasks efficiently 

## Description

The wireframes of User Story 1   –  8  are the prototypes for this user story

## Acceptance criteria

- [ ] The design is more clearly arranged for the user:
    - [ ] The overview with a list of trips
    - [ ] The detailspage of a single trip with all information and a packing list
    - [ ] The form fields and their description 
    - [ ] The buttons 
    - [ ] The images
    - [ ] The headers

## Tasks

- [ ] Create the feature branch `feature/basicDesign`
- [ ] Add one breakpoint for a bigger device
- [ ] Add design elements for structuring the content (Overview page, detailspage and the associated forms)

**Comment by ahohnsen** [comment]
Heyho,

As already discussed, I would suggest to update this user story as the main purpose is to improve the UI for the user.

Suggestion for a more fitting value proposition:

_As a user, 
I want to see a more organized and user-friendly user interface (UI),
so that I can easily interact with the application and complete tasks efficiently._

**Acceptance Criteria**
You have developed already mobile first. If I understood correctly, you want your app to be also visually appealing on a bigger screen. Therefore I would rephrase the first point of your acceptance criteria.
Additionally, if there are specific elements you want to improve (e.g. a form etc.), I suggest to include them in your acceptance criteria as well.

**Tasks**
As already discussed, your app is working on a mobile screen. You would need to add breakpoints for bigger devices. 🙂

**Comment by JessicaLoers** [comment]
**Well done.** 👍

**Comment by shrocket** [comment]
Hello and thank you for letting me look at your work!

If I understand correctly, I was just to make sure "The design is more clearly arranged for the user," but that I should keep in mind that "packing list, alerts and 'edit a trip' aren't ready yet." These instruction are a bit confusing, but I can say the following:
- the overview with a list of trips looks good!
- the detailspage of a single trip with all information and a packing list looks good!

Please continue to make your app pretty! and usable! and accessible!

Happy coding!
